mainwindow.cpp

```diff
@@ -797,21 +799,26 @@ bool MainWindow::loadConfFile(const QString filename)
         while( !confFile.atEnd() )
         {
             enabledOption = true;
-            currentLine = confFile.readLine();
-            currentLine.remove(' ');
+            fullCurrentLine = confFile.readLine();

```

Comment:
Is there even need for this rename?  I think that you could keep the original name and there would be no problem.